### PR TITLE
fix: method POST must have a request body

### PR DESCRIPTION
**Pull request by andreaTP**
This is a fix for the issue found here:
https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/pull/502#discussion_r1136044789

```
// Unexpected exception thrown: java.lang.IllegalArgumentException: method POST must have a request body.
```

### CODE REVIEW

**Review by baywet — CHANGES_REQUESTED** [review]
Thanks for the contribution.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Fix add an empty body when method is POST and the body is null
```
[the remark this answers is not in the document]
Comment (andreaTP) [reply CHANGELOG.md:14]: amazing! thanks!

**Comment by andreaTP** [comment]
@baywet should be ready to go 🙂

**Review by baywet — DISMISSED** [review]
Thanks for making the updates.

**Comment by andreaTP** [comment]
@baywet I missed an android linting thing, should be good now 👍 Looking for the release, and thanks a lot!

**Review by baywet — APPROVED** [review]

**Comment by andreaTP** [comment]
> thanks for making the update, I didn't know you could put the attribute inline for the return type like you can for parameters.

Oh, sorry, I haven't noticed that you are using a different style in this codebase, do you want me to follow up on this?
Also, since we are here 🙂 , do you have any opinionated automatic formatting tool for Java already in place somewhere?

**Comment by baywet** [comment]
I personally don't care about those things.

But we have some rules in spotbugs that do a very basic level of linting, I think that's about it for the linting today.

If you have spare time, setting up something like checkstyles or equivalent would be appreciated.

**Comment by andreaTP** [comment]
> I personally don't care about those things.

Forgot about it 😅 Right!

> If you have spare time, setting up something like checkstyles or equivalent would be appreciated.

Deal, next time I jump on this codebase and feel bad because of the formatting I'll do it 🙂

**Pull request merged**